app/views/idv/phone/new.html.erb

```diff
@@ -64,7 +64,7 @@
  <%= f.input :phone, required: true, input_html: { aria: { invalid: false }, class: 'sm-col-8' }, label: false,
    wrapper_html: { class: 'margin-right-2' } %>
 
-  <% if FeatureManagement.enable_gpo_verification? %>
+  <% if @gpo_letter_available %>
```

Comment:
what if we inlined the `!Idv::GpoMail.new(current_user).mail_spammed?` logic here? then we wouldn't have to remember to set the variable in 3 places?

Reply:
The wrinkle is it includes at least one database query. I lean towards keeping the view as stateless as possible.

Reply:
generally I like to keep complex logic out of templates too, but in this case, it seems too easy to miss a new spot.

Maybe we could swap it to being a local variable so that the render blows up if we don't pass it explicitly?

Reply:
That sounds good to me!